### PR TITLE
i18n: Add Arabic

### PR DESCRIPTION
**Pull request by muhammadbahaa2001**
Arabic Translation for Saber.

### CODE REVIEW

**Review by adil192 — CHANGES_REQUESTED** [review]
There seem to be a few formatting errors with incorrect escape sequences scattered around the file. Please fix these and I'll take another look :)

**Comment by muhammadbahaa2001** [comment]
Sorry,  what do you mean exactly by "few formatting errors with incorrect escape sequences scattered around the file"? give an example from what I did, so I can fix it.

**Review by adil192 — CHANGES_REQUESTED** [review]
Also please can you correct the spelling of "Sabre" to "Saber"

Edit: This was done a few times @muhammadbahaa2001

File: lib/i18n/strings_ar.i18n.json [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    },
+    "accentColorPicker": {
+      "pickAColor": "اختر لوناً"
+    },
+    "straightenDelay": {
+      "off": "مغلق",
+      "regular": "عادي",
+      "slow": "بطيء"
+    },
+    "systemLanguage": "System language",
```
Comment (adil192) [inline lib/i18n/strings_ar.i18n.json:75]: Looks like you forgot this line!

**Review by adil192 — CHANGES_REQUESTED** [review]

**Comment by muhammadbahaa2001** [comment]
> Also please can you correct the spelling of "Sabre" to "Saber"
> 
> Edit: This was done a few times @muhammadbahaa2001

Sorry for my mistakes (For every "Sabre" I miss it)

**Comment by adil192** [comment]
> Sorry for my mistakes (For every "Sabre" I miss it), I wish everything is corrected now.

No worries!

**Review by adil192 — APPROVED** [review]
LGTM, thanks for your contribution!

**Comment by adil192** [comment]
Would you be interested in being mentioned when there's new strings that need to be translated? https://github.com/adil192/saber/issues/19#issuecomment-1306096416

**Comment by muhammadbahaa2001** [comment]
> Would you be interested in being mentioned when there's new strings that need to be translated? [#19 (comment)](https://github.com/adil192/saber/issues/19#issuecomment-1306096416)

Of course, why not?

**Comment by muhammadbahaa2001** [comment]
> LGTM, thanks for your contribution!

You are welcome bro :)

**Pull request merged**